libraries/cmake/source/libdevmapper/config/configure.h

```diff
@@ -479,9 +476,16 @@
 /* Define to 1 if you have the `strtoull' function. */
 /* #undef HAVE_STRTOULL */
 
+/* Define to 1 if `st_blocks' is a member of `struct stat'. */
+#define HAVE_STRUCT_STAT_ST_BLOCKS 1
+
 /* Define to 1 if `st_rdev' is a member of `struct stat'. */
 #define HAVE_STRUCT_STAT_ST_RDEV 1
 
+/* Define to 1 if your `struct stat' has `st_blocks'. Deprecated, use
+   `HAVE_STRUCT_STAT_ST_BLOCKS' instead. */
+#define HAVE_ST_BLOCKS 1
```

Comment:
Do we still need this, judging from the comment it looks deprecated? We already have `#define HAVE_STRUCT_STAT_ST_BLOCKS 1` above.